### PR TITLE
Fixed Arch Linux Dependencies

### PR DESCRIPTION
**Pull request by comrumino**

### CODE REVIEW

File: install-build-deps.sh [hunk 1]
```diff
@@ -63,9 +63,9 @@ fi
 
 # Install dependencies for Arch Linux
 if [ -e /usr/bin/pacman ] ; then
-  echo "Checking arch package provides and installed packages" "cmake"
-  declare -a arr=("zsync" "git" "libarchive" "autoconf" "libtool" "make"
-    "libtool" "fuse2" "xz" "glib2" "openssl" "inotify-tools" "lz4" "gcc" "g++")
+  echo "Checking arch package provides and installed packages"
```
Comment (TheAssassin) [inline install-build-deps.sh]: libtool is listed twice.

Comment (comrumino) [reply install-build-deps.sh]: done

**Comment by TheAssassin** [comment]
Can you add `g++` to the list again? There's a chance we add C++ stuff in the future. It's installed for the other distros as well.

**Comment by comrumino** [comment]
g++ is included in gcc, per
```
[archie@darchie ~]$ pacman -Qo g++
/usr/bin/g++ is owned by gcc 7.3.1+20180312-2
```
More information is also available in the [PKGBUILD](https://git.archlinux.org/svntogit/packages.git/tree/trunk/PKGBUILD?h=packages/gcc).

**Comment by TheAssassin** [comment]
Thanks for the fixes.

**Pull request merged**